.jujuignore

```diff
@@ -0,0 +1,8 @@
+/env
+/image
+.tox
+*.py[cod]
+*.charm
+image-builder
```

Comment:
What's this `image-builder`?

Reply:
Removed the unused image-builder! Thank you!